Source/Utils/ParticleUtils.cpp

```diff
@@ -57,12 +57,9 @@ namespace ParticleUtils
         ParticleBins bins;
         bins.build(np, ptd, cbx,
             // Pass lambda function that returns the cell index
-            [=] AMREX_GPU_DEVICE (ParticleType const & p) noexcept -> amrex::IntVect
+            [=] AMREX_GPU_DEVICE (ParticleType const & p) noexcept -> IntVect
             {
-                return IntVect{AMREX_D_DECL(
-                                   static_cast<int>((p.pos(0)-plo[0])*dxi[0] - lo.x),
-                                   static_cast<int>((p.pos(1)-plo[1])*dxi[1] - lo.y),
-                                   static_cast<int>((p.pos(2)-plo[2])*dxi[2] - lo.z))};
+                return getParticleCellIndex(p, plo, dxi, lo);
```

Comment:
It seems this change is causing some crashes in code that uses the particle binning. I'm thinking it is due to the dimension specific functionality in `getParticleCellIndex` that is absent here.
I think this can still be simplified though, using
```suggestion
                return amrex::getParticleCell(p, plo, dxi, cbx);
```
and then the `lo` variable would not be needed anymore.